src/wallpaper.c

```diff
@@ -305,15 +305,19 @@ void feh_wm_set_bg(char *fil, Imlib_Image im, int centered, int scaled,
 
 		D(("Falling back to XSetRootWindowPixmap\n"));
 
+		XColor color;
+		Colormap cmap = DefaultColormap(disp, DefaultScreen(disp));
+		if (opt.image_bg)
```

Comment:
Could `image_bg` default to `"black"` actually to remove this branch?

Reply:
We can do it like that, if we drop the old 'default' behavior:
> The default for windowed mode is checks, while fullscreen defaults to black.